### ISSUE TITLE
Cannot Borrow 'query_selectables' as mutable, as it is not declared as mutable

### ISSUE THREAD

**Issue by UnFlimFlammable** [aevyrie/bevy_mod_picking#32]
With updated versions of Bevy / Cargo I get this error:

```
error[E0596]: cannot borrow `query_selectables` as mutable, as it is not declared as mutable
   --> lib.rs:316:39
    query_selectables: Query<&SelectablePickMesh>,
  ----------------- help: consider changing this to be mutable: `mut query_selectables`

```
cargo.toml dependencies:

```
[dependencies]
bevy = { git = "https://github.com/bevyengine/bevy", rev = "34c6f5f41bd2bff7867e269985a54036b8d96771"}
bevy_mod_picking = { git = "https://github.com/aevyrie/bevy_mod_picking", branch = "master" }
```

Appears to be a breaking change from Bevy. 

Rustup / Cargo Info:
```
active toolchain
----------------

nightly-x86_64-pc-windows-msvc (default)
rustc 1.48.0-nightly (d006f5734 2020-08-28)
```

**Comment by aevyrie** [comment]
Looks like a bevy commit that changed the inner machinations of the ECS system caused this.
https://github.com/bevyengine/bevy/commit/70ad6671dba9e7dc93e8a085ad8bb73904ca1fa5

**Comment by aevyrie** [comment]
Fixed with PR #35 !
